functions/routes/index.js

```diff
@@ -139,6 +117,65 @@ router.post("/worker", async function(req, res, next) {
     }
 });
 
+router.post("/twimlNumber", async function (req, res) {
+    const { Caller } = req.body;
+    const phone = Caller.substring(2);
+
+    res.writeHead(200, { "Content-Type": "text/xml" });
+    //const resp = `<?xml version="1.0" encoding="UTF-8"?><Response><Say>Is<say-as interpret-as="digits">${phone}</say-as>the best number we can reach you at?</Say><Redirect>https://webhooks.twilio.com/v1/Accounts/ACb228c71773482b13000655101442e779/Flows/FWf23aac20d25b198078c9b6c98957da34?FlowEvent=return</Redirect></Response>`;
+    const resp = twilioService.generateTwiML(
+        "Is",
+        phone,
+        "the best number we can reach you at?",
+        process.env.TWILIO_STUDIO_RETURN_URL
+    );
+    return res.end(resp.toString());
+});
+
+router.get("/twimlZipcode", async function (req, res) {
+    const { zipcode } = req.query;
+    res.writeHead(200, { "Content-Type": "text/xml" });
```

Comment:
why do you send the response in XML?

Reply:
Twiml requires an XML of it's markup language.